source/reference/minio-server/settings/console.rst

```diff
@@ -175,8 +175,9 @@ Defaults to ``default-src 'self' 'unsafe-eval' 'unsafe-inline';``
          .. code-block:: shell
             :class: copyable
 
-            set MINIO_BROWSER_CONTENT_SECURITY_POLICY="default-src 'self' 'unsafe-eval' 'unsafe-inline';"
+            export MINIO_BROWSER_CONTENT_SECURITY_POLICY="default-src 'self' 'unsafe-eval' 'unsafe-inline';"
```

Comment:
Is there a `windows` | `not windows` differentiator for this text?

Reply:
no, we don't produce windows docs for the Minio Server reference docs. They're linux only.